jsapp/js/components/permissions/publicShareSettings.component.tsx

```diff
@@ -22,6 +22,7 @@ interface PublicShareSettingsProps {
   publicPerms: PermissionResponse[];
```

Comment:
There are four unused imports on the top of this file (some leftovers from other PR), could you please remove them? :)